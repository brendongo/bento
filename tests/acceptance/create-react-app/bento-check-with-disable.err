Running Bento checks...


⚠ 176 finding(s) in  s
◦ To suppress all findings, run $ bento archive.
