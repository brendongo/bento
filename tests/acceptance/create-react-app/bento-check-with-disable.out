docusaurus/website/src/pages/index.js
  9:18 error   Unable to resolve path to module '@docusaurus/Link'.                                                 r2c.eslint     import/no-unresolved https://eslint.org/docs/rules/import/no-unresolved
 10:34 error   Unable to resolve path to module '@docusaurus/useDocusaurusContext'.                                 r2c.eslint     import/no-unresolved https://eslint.org/docs/rules/import/no-unresolved
 11:25 error   Unable to resolve path to module '@docusaurus/withBaseUrl'.                                          r2c.eslint     import/no-unresolved https://eslint.org/docs/rules/import/no-unresolved
 13:20 error   Unable to resolve path to module '@theme/Layout'.                                                    r2c.eslint     import/no-unresolved https://eslint.org/docs/rules/import/no-unresolved
 14:23 error   Unable to resolve path to module '@theme/CodeBlock'.                                                 r2c.eslint     import/no-unresolved https://eslint.org/docs/rules/import/no-unresolved
 16:24 error   Unable to resolve path to module 'classnames'.                                                       r2c.eslint     import/no-unresolved https://eslint.org/docs/rules/import/no-unresolved
 71:24 error   Do not use Array index in keys                                                                       r2c.eslint     react/no-array-index-key https://eslint.org/docs/rules/react/no-array-index-key

tasks/compile-lockfile.js
 12:13 error   'fs-extra' should be listed in the project's dependencies, not devDependencies.                      r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies
 27:24 error   'react-scripts' should be listed in the project's dependencies. Run 'npm i -S react-scripts' to a... r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies
 27:24 error   Unexpected require().                                                                                r2c.eslint     global-require https://eslint.org/docs/rules/global-require

tasks/screencast-start.js
 12:15 error   'execa' should be listed in the project's dependencies, not devDependencies.                         r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies
 13:14 error   'meow' should be listed in the project's dependencies, not devDependencies.                          r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies
 14:20 error   'multimatch' should be listed in the project's dependencies, not devDependencies.                    r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies

tasks/screencast.js
 14:15 error   'execa' should be listed in the project's dependencies, not devDependencies.                         r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies
 15:15 error   'tempy' should be listed in the project's dependencies, not devDependencies.                         r2c.eslint     import/no-extraneous-dependencies https://eslint.org/docs/rules/import/no-extraneous-dependencies
 41:18 error   Calls to require() should use string literals                                                        r2c.eslint     import/no-dynamic-require https://eslint.org/docs/rules/import/no-dynamic-require
 41:18 error   Unexpected require().                                                                                r2c.eslint     global-require https://eslint.org/docs/rules/global-require
 54:5  error   Unnecessary catch clause.                                                                            r2c.eslint     no-useless-catch https://eslint.org/docs/rules/no-useless-catch

